configure.ac

```diff
@@ -20,11 +20,7 @@ AC_PATH_GENERIC(giblib, 1.2.3, [
   AC_SUBST(GIBLIB_CFLAGS) ],
```

Comment:
This must be removed too, look at my [PR](https://github.com/languidnights/scrot/pull/1/) that I sent you, you can merge it or copy and paste the changes that I indicate there

Reply:
Thank you; I merged the PR